### PR TITLE
Initial version of jsplitpkgscan Maven plugin - executes the creation of a file in the verify phase

### PR DESCRIPTION
**Pull request by karianna**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-site-plugin</artifactId>
+                <version>3.6</version>
+            </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-surefire-plugin</artifactId>
+                <version>2.20.1</version>
+            </plugin>
+
```
Comment (lasombra) [inline pom.xml]: Sorry for the nitpick but why the humongous list of unconfigured plugins?

Comment (karianna) [reply pom.xml]: Don't apologise!  This is part of being strict about reproducible builds.  Each version of Maven (3.5.2, 3.3.9, 3.x.x) all set the plugin versions differently in their parent, so this is a way of setting them to a known good version permanently

Comment (lasombra) [reply pom.xml]: Interesting approach. With that in mind, should we also move the versions to properties?

Comment (karianna) [reply pom.xml]: I tend to only do that if the version is going to be used in two places.  (e.g. Surefire and failsafe if we use that)

Comment (lasombra) [reply pom.xml]: Got it. So LGTM 👍 .

**Pull request merged**